src/app/clusters/mode-select-server/mode-select-server.cpp

```diff
@@ -155,7 +155,7 @@ void emberAfModeSelectClusterServerInitCallback(EndpointId endpointId)
                 {
                     if (onOffValueForStartUp && !onMode.IsNull())
                     {
-                        emberAfPrintln(EMBER_AF_PRINT_DEBUG, "ModeSelect: CurrentMode is overwritten by OnMode");
+                        ChipLogProgress(Zcl, "ModeSelect: CurrentMode is overwritten by OnMode");
```

Comment:
_DEBUG should be ChipLogDetail everywhere.

Reply:
That is what I tought at first, however println seemed to have defaulted to ChipLogProgress and ignored the `category` which is DEBUG in this case, so I redirected everything to progress.

will double-check.

Reply:
So code is:

```
void emberAfPrint(int category, const char * format, ...)
{
    if (format != nullptr)
    {
        va_list args;
        va_start(args, format);
        chip::Logging::LogV(chip::Logging::kLogModule_Zcl, chip::Logging::kLogCategory_Progress, format, args);
        va_end(args);
    }
}
```

where category seems completely ignored and everything goes to ZCL/Progress.

Reply:
Updated PR description.

lock app increase was due to `EMBER_AF_PRINT_ENABLE` being actually disabled and no prints being made.